layout.ld

```diff
@@ -81,21 +85,28 @@ SECTIONS {
         *(.text*)
         *(.rodata*)
         KEEP (*(.syscalls))
-        _etext = .;
         *(.ARM.extab*)
         . = ALIGN(4); /* Make sure we're word-aligned here */
+        _etext = .;
     } > FLASH =0xFF
 
+    /* Application stack */
+    .stack :
+    {
+        . = ALIGN(8);
```

Comment:
Why did you add the alignment on line 96? The code adds stack size then aligns, and I think this needs to match.

Reply:
Good catch. We will need something like,

```
. = ALIGN(8);
_sstack = .;
```
As we do in the kernel. I'll fix it in V2.

Reply:
Looks like we'll need an assertion instead.